source/uwp/AdaptiveCardTestApp/Expected/microsoft-teams.Tests.Action.DuplicateIds.json

```diff
@@ -1 +1 @@
-{"HostConfigHash":"5da68b8","CardHash":"e423716","Error":"Parsing card failed"}
\ No newline at end of file
+{"HostConfigHash":"1eae351","CardHash":"e423716","Error":"Parsing card failed"}
```

Comment:
Do you know why all these files changed?

Reply:
HostConfigs got updated to include more ContainerStyleDefinitions. 

---
In reply to: [247212915](https://github.com/Microsoft/AdaptiveCards/pull/2289#discussion_r247212915) [](ancestors = 247212915)